deployment/scripts/indexd/indexd_setup.sh

```diff
@@ -11,4 +11,4 @@ poetry install -vv --no-dev --no-interaction && poetry show -v
 python3 -m pip install cdispyutils
 popd
 
-/dockerrun.sh
```

Comment:
this file isn't used by the Dockerfile, right? b/c above this change this is doing some poetry stuff manually

Reply:
yes! I don't think we need it at all, so I deleted it! Nice catch!